packages/cf-style-const/src/variables.js

```diff
@@ -14,6 +14,20 @@ export default {
     seven: 'linear-gradient(left, #F69259, #FFDB6F)',
     eight: 'linear-gradient(left, #85CBA8, #FFDB6F)'
   },
+  color: {
+    rain: '#408BC9',
```

Comment:
If design starts defining colors in mocks with these names its probably fine but if they say blue and give us the hexcode for rain we're going to need to rework this.  I think it adds an unnecessary level of confusion in my opinion.